runtime/jvmti/jvmtiExtensionMechanism.c

```diff
@@ -1780,6 +1780,8 @@ jvmtiInternalGetStackTraceIteratorExtended(J9VMThread * currentThread, J9StackWa
 #ifdef J9VM_INTERP_NATIVE_SUPPORT 
 			if (walkState->jitInfo == NULL) {
 				frame_buffer->type = COM_IBM_STACK_FRAME_EXTENDED_NOT_JITTED;
+			} else if ((type & J9JVMTI_STACK_TRACE_MARK_INLINED_FRAMES) && (walkState->inlineDepth > 0)) {
```

Comment:
Please use the `J9_ARE_ANY_BITS_SET` macro rather than `&`.

Reply:
Done

By the way, this feedback was equally applicable to the initial revision, so we could have saved a round trip